src/redux/actions/transactions.ts

```diff
@@ -183,14 +186,14 @@ export function handlePolkadotTransactionEvents(
     incentivizedChannelContract
       .events
       .MessageDispatched({})
-      .on('data', handleChannelMessageDispatched);
+      .on('data', handleChannelMessageDispatched(Channel.INCENTIVIZED));
 
     // TODO: replace with incentivized channel?
     // eslint-disable-next-line no-unused-expressions
     basicChannelContract
```

Comment:
maybe we should listen to events only from one contract depending on `ACTIVE_CHANNEL` setting?

Reply:
yeh, that could be better. i was imagining a potential future context in which we allow the user to select a channel rather than it being in the config, so just left it in in case but it would be cleaner with only one active listener now. anyway, fine to leave in i think